test_unstructured/staging/test_base.py

```diff
@@ -209,7 +202,7 @@ def test_all_elements_preserved_when_serialized():
     assert base.convert_to_isd(base.isd_to_elements(isd)) == isd
 
 
-def test_serialized_deserialize_elements_to_json(tmpdir):
+def test_serialized_deserialize_elements_to_json(tmpdir: str):
```

Comment:
I've noticed we don't type hint as much in tests, but why remove this?

Reply:
Not sure what you mean? We added the type annotation `: str` to `tmpdir` here, so we increased the level of type annotation.

I look to have tests type-check clean on strict, same as the code. Maybe tests tend to start out with less type annotation but they should end up with all they need when I'm done :)

Reply:
I must have read this backwards when reviewing. Looks good :)